project/config/lgbcni/config/core.base_field_override.node.publication_page.promote.yml

```diff
@@ -1,15 +1,13 @@
-uuid: 24b31959-b898-468e-8d3d-418873cc0093
+uuid: 0ce60a08-fdc8-40b0-a4a0-3f6bf7a94fb0
 langcode: en
 status: true
 dependencies:
   config:
-    - node.type.publication
-_core:
-  default_config_hash: RZReXxGt6xMN6rIDO654C-FmUbPmLjNOLeddn_iFPTc
-id: node.publication.promote
+    - node.type.publication_page
+id: node.publication_page.promote
 field_name: promote
 entity_type: node
-bundle: publication
```

Comment:
If you had of changed the machine name of the destination content type to publication, like here https://github.com/dof-dss/Unity_svr_3/blob/development/project/sites/judiciaryni/modules/custom/judiciaryni_migrations/config/install/migrate_plus.migration.upgrade_d7_node_complete_publication_page.yml#L157, you could have save yourself alot of work.

Reply:
Thanks, was thinking I could have done this after creating the PR unfortunately

Reply:
It changes all you display and form views if you do it after so better off doing it before you start the migration of content.
When you migrate in the content types be sure to delete to right one :)